src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Color {
+    blue,
+    green,
+    white,
```

Comment:
```suggestion
    BLUE,
    GREEN,
    WHITE,
```
enum elements should be written using CAPS